pkgs/build-support/flatten-references-graph/default.nix

```
@@ -0,0 +1,29 @@
+{ bash
+, nix-gitignore
+, python3Packages
+, writers
+}:
+let
+  helpers = import ./helpers.nix { inherit bash python3Packages writers; };
+  pythonPackages = python3Packages;
+
+in
+pythonPackages.buildPythonApplication {
+  version = "0.1.0";
+  pname = "flatten-references-graph";
+
+  # Note: this uses only ./src/.gitignore
+  src = nix-gitignore.gitignoreSource [ ] ./src;
+
+  propagatedBuildInputs = with pythonPackages; [
+    python-igraph
+    toolz
+  ];
+
+  doCheck = true;
+
+  checkPhase = ''
+    ${helpers.lint}/bin/lint
+    ${helpers.unittest}/bin/unittest
+  '';
```

```suggestion
  helpers = callPackage ./helpers.nix { };
in
python3Packages.buildPythonApplication {
  pname = "flatten-references-graph";
  version = "0.1.0";

  # Note: this uses only ./src/.gitignore
  src = nix-gitignore.gitignoreSource [ ] ./src;

  propagatedBuildInputs = with python3Packages; [
    python-igraph
    toolz
  ];

  doCheck = true;

  checkPhase = ''
    ${helpers.unittest}/bin/unittest
  '';
```
we never want to run linters to make sure the build does not suddenly fail because of some whitespace change

---

Ok, happy to disable it

---

Done in https://github.com/adrian-gierakowski/nixpkgs/commit/590b1e1dfe978c41075d0d9759b5f2b6f0d3859d